### DISCUSSION TITLE
Background Image not loading on multiple canvas at same time

### DISCUSSION THREAD

**Discussion by EssDevUi** [fabricjs/fabric.js#8451] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [x] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

4.5.0

### In What environments are you experiencing the problem?

Firefox, Chrome, Safari, Opera, Microsoft Edge

### Link To Reproduction

https://modconreg.com/

### Steps To Reproduce

1. Login to website (email : mcraccessoruser@yopmail.com , password : Ess12345)
2. goto view Records from Sidebar Accessor tab
3. sort by created to get the latest record
4. click on edit (pencil icon) Or view (eye icon)
5. scroll to bottom and click on next btn and goto 3rd step (modifications tab)
6. scroll down to see the multiple canvas
7. 
8.

 

### Expected Behavior

I'm using canvas.backgroundImage to load the images first then adding the shapes in it 
the images should load and shapes should be on their proper placements
as we have multiple canvas instances so we are itrating through each canvas instance to load it's expected image in it using custom JSON.

### Actual Behavior

https://www.loom.com/share/80231eb1aa614961881e2ae60102141d
as you can see sometime the image loads but sometimes it doesn't load it's shapes because shapes coordinates are dependent on 
background image 
![image](https://user-images.githubusercontent.com/86969405/202624213-beb297b8-b8c6-4c5a-8081-c772355bf1aa.png)
when the image take time to load the function breaks and generate an error 
which doesn't load the shapes in the Image 
![image](https://user-images.githubusercontent.com/86969405/202624424-f979a0ed-daa5-40c1-923c-66bcfb4d7750.png)

(On View page)
We are loading the canvas in (display:none) div and on page load after loading the canvas we are generating their SVG and showing in an div on runtime . 
this also do the same thing when the canvas image isn't loaded the svg doesn't created
![image](https://user-images.githubusercontent.com/86969405/202623550-9cb82bef-98fc-427b-b361-230dc5b332bb.png)

**Comment by ShaMan123** [comment]
This is a question, not a bug
More related to caching then to fabric if I understand correctly.
I suggest you post more concise issues if you want anyone to look into them. I personally didn't
